packages/components/package.json

```diff
@@ -13,7 +13,8 @@
     "@styled-system/color": "^5.1.2",
     "@styled-system/css": "^5.0.23",
     "@styled-system/should-forward-prop": "^5.1.2",
-    "@styled-system/space": "^5.1.2"
+    "@styled-system/space": "^5.1.2",
+    "deepmerge": "^4.2.2"
```

Comment:
You can use the `merge` utility from `@theme-ui/core` for this instead

Reply:
I tried this out but it broke the code. I've switched back to deepmerge for the time being.